### PR TITLE
adding exercise descriptions and setup notes

### PR DESCRIPTION
**Pull request by CraigGosnay**
Not completed yet. Just creating the PR to share

### CODE REVIEW

File: site/src/pages/setup.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import React from "react"
+import { graphql } from "gatsby"
+
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+const SetupPage = ({ data, location }) => {
+  const siteTitle = data.site.siteMetadata.title
+
+  return (
```
Comment (CraigGosnay) [inline site/src/pages/setup.js:10]: We need to talk about how I'm supposed to do this properly as this clearly isn't the way, and I haven't had the time to research it.

Comment (mimiflynn) [reply site/src/pages/setup.js:10]: Do which part properly? I've been underwater with other work. Thanks for taking time to do any of this at all. Its immensely helpful.

Comment (CraigGosnay) [reply site/src/pages/setup.js:10]: I'm not sure the whole page in html in one chunk is the 'react / gatsby' approach :) 

Yeah no problem. I hope to have some time soon to actually learn a bit of this paradigm, just want to get the content in so we arn't in a rush for the July event.

File: site/src/pages/setup.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      <div style={{ marginTop: `30px`, fontFamily: `Montserrat` }}>
+
+        <h1>Start Here</h1>
+        <h3>What is the Circuit Playgroud Express?</h3>
+        <p>The Circuit Playgroud Express is an all-in-one circuit board 
+            featuring a processor, sensors, LEDs, USB and more!
+        </p>
+
+        <h3>In Detail</h3>
+        <div class="container">
```
Comment (mimiflynn) [inline site/src/pages/setup.js:22]: React requires `className` instead of `class`.

**Pull request merged**